generic/tests/cfg/vlan.cfg

```diff
@@ -14,14 +14,14 @@
     kill_vm_gracefully_vm2 = no
     cmd_type = ip
     msg_pattern = '8021q: Invalid VLAN id'
-    RHEL.7, RHEL.6:
+    RHEL.7, ALT.7, RHEL.6:
         msg_pattern = 'RTNETLINK answers: Numerical result out of range'
     Windows:
         # two windows VM, need more time to boot and login
         login_timeout = 720
         vlan_num = 1
         win_vlan_id = 900
-        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID ${win_vlan_id} -Confirm:$false"
+        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID '%s' -Confirm:$false"
```

Comment:
Is there any difference?

Reply:
the python script did not use `win_vlan_id` parameter before.
```
        win_vlan_id = int(params.get("win_vlan_id", 900))
        set_vlan_cmd = params.get("set_vlan_cmd")
        set_vlan_cmd = set_vlan_cmd % nicid
```

Reply:
@PaulYuuu Yes, so that should be removed from the script.

Reply:
If I remove it from the script, where does it get the value of `win_vlan_id`? Also from `vlan.cfg`?

Reply:
@PaulYuuu Yes, the original vlan.cfg `${win_vlan_id}`, that's the grammar of Cartesian configuration